Project-Descriptions-and-Plans/CVA6/CVA6-main-plan-approved-2024+.md

```diff
@@ -0,0 +1,524 @@
+| Cores           | Define and add memory protection for harsh environments (error detection and correction)                                       | 2024          | Interested stakeholders                       |
+| Cores           | Integrate in-order dual-issue                                                                                                  | 2024-Q2       | TSS                                           |
+| Cores           | Integrate instruction and data scratchpads                                                                                     | 2024-Q2       | TRISTAN partner                               |
+| Cores           | Parametrization (replace configuration packages and `directives by top-level SV parameters)                                    | 2024-Q2       | TSS                                           |
+| Cores           | Evaluate HPDCache as L1 data cache, decide if it's the project default L1 date cache                                           | 2024-Q1       | Interested stakeholders                       |
+| Cores           | _To be confirmed:_ Upgrade CV-X-IF to the newly ratified version                                                               |               | TSS                                           |
+| Cores           | _To be confirmed:_ Evolutions for the tightly-coupled cache coherence project                                                  |               | PlanV                                         |
+| Cores           | _To be confirmed:_ Configurable reset                                                                                          |               | ECC                                           |
+| Verification    | Maintain GitHub-triggered CI flow                                                                                              | Continuous    | TSS                                           |
+| Verification    | Maintain tandem mode                                                                                                           | Continuous    | OpenHW staff                                  |
+| Verification    | Add Mentor Questa support to the verification flow                                                                             | 2024          | OpenHW staff                                  |
+| Verification    | Add Cadence Xcelium support to the verification flow                                                                           | 2024          | TRISTAN partner                               |
+
+### Project tracking and meetings
+
+#### Project tracking
+
+The progress is tracked in CVA6 meetings and sub-project meetings, with a joint GitHub based [Kanban board](https://github.com/orgs/openhwgroup/projects/3).
+Labels are used to differentiate configuration-specific tasks.
+
```

Comment:
Great for the Kanban board. Could we have a monthly or regular cadence project update report using the [simple project report template](https://github.com/openhwgroup/programs/blob/master/process/Project-Monthly-Report-Template.md) in the [cva6 reporting directory](https://github.com/openhwgroup/programs/tree/master/TGs/cores-task-group/cva6) to help people locate the status? Another alternative is to update monthly slides in that directory.

Reply:
I'll add this in the document. And I need to do it every month :-)